resin/exceptions.py

```diff
@@ -2,222 +2,204 @@
 from .resources import Message
 
 
-class MissingOption(Exception):
+class ResinException(Exception):
```

Comment:
Adding a `ResinException` baseclass obviously isn't strictly related to adding Python3, but it's nice to see anyway. This change should probably be mentioned in the commit-message?

Reply:
my bad, I've updated the commit message to clarify this.